.github/workflows/dependency-updates-post-processing.yaml

```diff
@@ -55,12 +56,13 @@ jobs:
       - name: Make devbox shellenv available
         run: |
           eval "$(devbox shellenv)"
-          printenv >> $GITHUB_ENV
+          printenv >> "${GITHUB_ENV}"
 
       - name: go-mod-tidy and make package-specs
         shell: bash
         env:
-          GH_TOKEN: ${{ secrets.APP_AUTOSCALER_CI_TOKEN }}
+          GH_TOKEN: "${{ secrets.APP_AUTOSCALER_CI_TOKEN }}"
+          PR_BRANCH_REF: "${{ github.head_ref }}"
```

Comment:
Introducing this via environment-variable improves security of this workflow (against code-injection).